src/components/text/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+import { Text as ChakraText, TextProps } from '@chakra-ui/react';
+
+type MarginProps = 'm' | 'mt' | 'mb' | 'mr' | 'ml' | 'my' | 'mx';
```

Comment:
could we use `SpaceProps` or do you want to avoid that devs pass marginX instead of mx?

Reply:
Yeah, I wanted to harmonise the syntax we use here.